### DISCUSSION TITLE
crazyflie and flapper

### DISCUSSION THREAD

**Discussion by prathyushaavs** [bitcraze/discussions#1422] [Q&A]
hey can you provide the code to make a flapper and crazyflie fly together?

**Comment by tobbeanton** [comment]
You would have to be more precise in your question. E.g. are you using a positioning system?

**Reply by prathyushaavs** [reply]
thank you for your reply
i am using lighthouse positioning system

**Reply by tobbeanton** [reply]
I suggest you to check out the cflib [examples](https://github.com/bitcraze/crazyflie-lib-python/tree/master/examples). First get each of them to fly and then you could combine them using one of the swarm examples as a base.

**Reply by prathyushaavs** [reply]
i have tried them flying individually.

**Reply by tobbeanton** [reply]
Have you tried altering any of the swarm examples?

**Reply by prathyushaavs** [reply]
no i didnt alter.

**Reply by tobbeanton** [reply]
Then I suggest you to do that.

**Reply by matejkarasek** [reply]
Yes, when using the Python API (cflib), the flapper should behave like a  crazyflie, i.e. just specify its uri address/channel/.... Any of the swarm examples should work.
